cherrypy/process/wspbus.py

```diff
@@ -342,15 +342,10 @@ def block(self, interval=0.1):
             # that another thread executes cherrypy.engine.exit()
             if (
                     t != threading.currentThread() and
-                    t.isAlive() and
                     not isinstance(t, threading._MainThread)
```

Comment:
Okay, it looks since `_DummyThread`s are always daemonic additional check won't affect the flow in any way. Let's leave it as is. Also I've noticed that we can now merge `if` blocks